tests/monitoring/monitoring.go

```diff
@@ -392,11 +464,13 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 			if !errors.IsAlreadyExists(err) {
 				util.PanicOnError(err)
 			}
-			revertScale(virtOperator.deploymentDame)
+			revertScale(virtOperator.deploymentName)
 
 			time.Sleep(10 * time.Second)
-			waitUntilAlertDoesNotExist("VirtOperatorDown")
-			waitUntilAlertDoesNotExist("NoReadyVirtOperator")
+			waitUntilAlertDoesNotExist(virtOperator.downAlert)
```

Comment:
I see that you'e omitted `waitUntilAlertDoesNotExist("NoReadyVirtOperator")`, is that in purpose?

Reply:
in this set of tests, I think it's okay to skip that verification